spec/duration.html

```diff
@@ -438,10 +479,35 @@ <h1>ToTemporalDurationRecord ( _temporalDurationLike_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-durationsign" aoid="DurationSign">
+      <h1>DurationSign ( _years_, _months_, _weeks_, _days_, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_ )</h1>
+      <emu-alg>
+        1. For each value _v_ of _years_, _months_, _weeks_, _days_, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_, do
```

Comment:
```suggestion
        1. For each value _v_ of « _years_, _months_, _weeks_, _days_, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_ », do
```

Reply:
I'm not sure what the correct spec language is, but there seems to be a precedent for not using the angle brackets: https://tc39.es/ecma402/#sec-todatetimeoptions